dbt/models/reporting/reporting.vw_pin_most_recent_sale.sql

```diff
@@ -4,9 +4,14 @@
 WITH all_pins AS (
     SELECT DISTINCT parid
     FROM {{ source('iasworld', 'pardat') }}
-    WHERE taxyr = CAST(YEAR(CURRENT_DATE) AS VARCHAR)
-        AND cur = 'Y'
+    WHERE cur = 'Y'
         AND deactivat IS NULL
+        -- noqa: disable=RF02
```

Comment:
There may be a bug in [RF02](https://docs.sqlfluff.com/en/stable/reference/rules.html#rule-references.qualification) in our version of SQLFluff, since this block doesn't seem to me to represent an unqualified reference. It doesn't raise an error, but the warning is annoying, so I ignore it just for this block.